data/brands/tourism/hotel.json

```
@@ -317,6 +328,28 @@
         "tourism": "hotel"
       }
     },
+    {
+      "displayName": "citizenM",
+      "id": "citizenm-779ccb",
+      "locationSet": {"include": ["ch", "dk", "fr", "gb", "it", "my", "nl", "tw", "us"]},
+      "tags": {
+        "brand": "citizenM",
+        "brand:wikidata": "Q5122531",
+        "name": "citizenM",
+        "tourism": "hotel"
+      }
+    },
+    {
+      "displayName": "City Express",
+      "id": "cityexpress-b84op3",
+      "locationSet": {"include": ["cl", "cr", "do", "mx"]},
```

```suggestion
      "locationSet": {"include": ["cl", "cr", "co", "mx"]},
```
Based on their website it looks like they have locations in Colombia and not the Dominican Republic